### PR TITLE
ReadMe Presentation/Documentation 

### PR DESCRIPTION
**Pull request by Vinyzu**
- TODO: Create Video Presentation and update YouTube Link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
 Did you ever want to experience the information overload and stress of a phone operator?
 Well, then you're at the right place!
 
-We've created a first stressful Discord game, in which you play a phone operator for a restaurant.
+We've created the first stressful Discord game, in which you play a phone operator for multiple restaurants.
```
Comment (koviubi56) [inline README.md:12]: That's a little too much

File: README.md [hunk 2]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+6. Run The Game:
    ```shell
    python -m sincere_singularities
    ```
-6. Play!
+7. Play!
+
+## Video Presentation
+
+Disclaimer: `The video contains Spoilers`
```
Comment (koviubi56) [inline README.md:48]: duh

File: README.md [hunk 3]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
-4. Set the `BOT_TOKEN` environment variable to your Token using the `.env` file.
-5. Run The Game:
+4. Setup and run [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/)
+5. Set the `BOT_TOKEN`, `DB_HOST`, `DB_PORT` and `DB_NAME` environment variables using the `.env` file.
+6. Run The Game:
    ```shell
    python -m sincere_singularities
    ```
-6. Play!
+7. Play!
```
Comment (koviubi56) [inline README.md:44]: Why is this here if it is repeated below?

File: README.md [hunk 4]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+
+
+## Installation
+<details>
+    <summary>Detailed Installation Guide ⚙️</summary>
+
+### 1. Requirements:
+   1. [Python 3.11](https://www.python.org/downloads/release/python-3110/)
+   2. [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/). See [MongoDB Installation Guide]
+   3. [Discord Bot](https://docs.disnake.dev/en/stable/discord.html). See [Discord Bot SetUp Guide]
```
Comment (koviubi56) [inline README.md:63]: These `[Discord Bot SetUp Guide]` links don't work. Headings make anchors, so you can link to them like this: `[Discord bot setup guide](#discord-bot-installation-guide)`

File: README.md [hunk 5]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+### 2. Download:
+Run this command in the directory you want to download it to.
+   ```shell
+   git clone https://github.com/SincereSingularities/SincereSingularities/
+   cd SincereSingularities
+   ```
+### 3. Install the Game as a PIP package & install requirements:
```
Comment (koviubi56) [inline README.md:70]: pip already installs the requirements. Either remove the reference or make it clear that pip already does that.

File: README.md [hunk 6]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+   ```shell
+   pip install -e .
+   ```
+### 4. Setup local environment values:
+Create & Edit an .env file (see .env.example)
```
Comment (koviubi56) [inline README.md:75]: Could we just use `and` instead of `&` everywhere?

File: README.md [hunk 7]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+```
+BOT_TOKEN (Your Discord Bot Token)
+DB_HOST (The IP address of your MongoDB Server)
+DB_PORT (The port of your MongoDB Server)
+DB_NAME (Your preffered name for the MongoDB Database)
```
Comment (koviubi56) [inline README.md:80]: State that it defaults to `bot_db`

File: README.md [hunk 8]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
 1. Choose a text channel.
 2. Run the `/start_game` command. That will create a thread.
 3. The bot will send the menu. This contains multiple restaurants that you can buy. You already own the first one.
 4. You will get orders in the thread as messages. Choose and enter the appropriate restaurant, and select the menu items that the customer requested.
 5. Then enter the customer's information which consists of:
+- Order ID
+- Customer Name
+- Customer Address
+- Delivery Time
+- Extra Wishes
```
Comment (koviubi56) [inline README.md:113]:
```suggestion
  - Order ID
  - Customer Name
  - Customer Address
  - Delivery Time
  - Extra Wishes
```

File: README.md [hunk 9]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+6. Enter the ordered items.
```
Comment (koviubi56) [inline README.md:114]: That's already stated in 4.

File: README.md [hunk 10]
```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+7. Submit the order. You will get points based on the accuracy of the order. With your earned coins you can buy new restaurants.
+8. From time to time, you will get Order Conditions. Pay attention to these Conditions when fulfilling a order. They will also disappear after some time.
```
Comment (koviubi56) [inline README.md:116]: Better explanation maybe? What exactly are they? What should you do? They only affect certain restaurants. When the message disappears, it's no longer active. ...

File: README.md [hunk 11]
```diff
@@ -9,31 +9,153 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
-## Running the bot
+## Table of contents
+1. [Getting Started](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#getting-started)
+2. [Video Presentation](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#video-presentation)
+3. [Installation](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#installation)
+4. [Running](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#running)
+5. [Gameplay Tutorial](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#gameplay)
+6. [Discord Bot Installation Guide](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#discord-bot-installation-guide)
+7. [MongoDB Community Edition Installation Guide](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#mongodb-community-edition-installation-guide)
+8. [Credits](https://github.com/SincereSingularities/SincereSingularities?tab=readme-ov-file#credits)
```
Comment (koviubi56) [inline README.md:24]: GitHub already has this feature.

**Review by WassCodeur — APPROVED** [review]
LGTM

File: README.md [hunk 12]
```diff
@@ -9,31 +9,153 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
...
+   1. For Installation, follow [Debian Installation Guide](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-debian/#install-mongodb-community-edition)
+   2. To Run MongoDB, follow [Debian Running Guide](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-debian/#run-mongodb-community-edition)
+5. Install on SUSE
+   1. For Installation, follow [SUSE Installation Guide](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-suse/#install-mongodb-community-edition)
+   2. To Run MongoDB, follow [SUSE Running Guide](https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-suse/#run-mongodb-community-edition)
+
+</details>
+
+# Credits
+This project was created by (in order of contributed LOC):
```
Comment (koviubi56) [inline README.md:157]: Hmm, maybe alphabetical could be better. You're _kinda_ implying that they matter less.

**Review by koviubi56 — CHANGES_REQUESTED** [review]
There are a few stuff that could be improved.

File: README.md (hunk 1 above)
Comment (Vinyzu) [reply README.md:12]: Could you eloborate?

Comment (koviubi56) [reply README.md:12]: We probably didn't create the _first_ stressful Discord game.

Comment (koviubi56) [reply README.md:12]: Or did you mean that we created the first stressful Discord game _where you are a phone operator for multiple restaurants_? Then it's probably fine (if it's true).

File: README.md (hunk 3 above)
Comment (Vinyzu) [reply README.md:44]: This is a quick guide. Its to give the user an overview on whats needed to run the bot, and can be used to expand/translate the installation to the users preferred methods (e.g. a package manager which isnt `pip`).

A more complete installation/running guide can be seen below. This is a step-by-step guide, which you cant do anything wrong with.

File: README.md (hunk 6 above)
Comment (Vinyzu) [reply README.md:75]: Used it once 😅

File: README.md (hunk 3 above)
Comment (koviubi56) [reply README.md:44]: Ok. Maybe you could also link to the appropriate sections.

File: README.md (hunk 6 above)
Comment (koviubi56) [reply README.md:75]: (twice actually: `Install the Game as a PIP package & install requirements`)

File: README.md (hunk 12 above)
Comment (Vinyzu) [reply README.md:157]: In the ReadMe, I took heavy inspiration from my last [CodeJam](https://github.com/Kingly-elpies/KinglyKelpies/tree/main?tab=readme-ov-file#credits).
But i also looked at my first [CodeJam](https://github.com/cj8-cheerful-cheetahs/Python-Code-Jam-2021-Project?tab=readme-ov-file#credits).

Both ReadMe's handled the credits like this `(in order of contributed LOC)` (not explicitly stated).
But if you believe alphabetical order would be fairer though, its ofc ok!

Comment (koviubi56) [reply README.md:157]: It's fine either way.

File: README.md (hunk 10 above)
Comment (Vinyzu) [reply README.md:116]: IMO, the Gameplay Explanation should only explain the game in its basis. If you want to learn how the Order Conditions actually work, either play the game yourself (learning-by-doing) or whatch the video presentation.

Comment (koviubi56) [reply README.md:116]: Ok. :+1:

File: README.md (hunk 1 above)
Comment (Vinyzu) [reply README.md:12]: Thats what i meant. The sentence is also structured in such way that this interpretation makes gramaticly sense.
I ofc cannot say with 100% certainty that its actually the first, but its extremely likely.

Also, a bit of "clickbait" wont hurt 😋.

Comment (koviubi56) [reply README.md:12]: 👌

File: README.md (hunk 12 above)
Comment (Vinyzu) [reply README.md:157]: Ok, then ill leave it like that because i dont wanna spend 2 minutes rearranging the table xD

File: README.md (hunk 3 above)
Comment (Vinyzu) [reply README.md:44]: Sure.

File: README.md (hunk 11 above)
Comment (Vinyzu) [reply README.md:24]: Could you explain/link a guide/documentation for this?

Comment (koviubi56) [reply README.md:24]: [Here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#auto-generated-table-of-contents-for-readme-files) and [here](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)

Comment (Vinyzu) [reply README.md:24]: OK! So just remove this section?

Comment (koviubi56) [reply README.md:24]: Yep.
